test/cypress/integration/element-targeting.spec.js

```diff
@@ -9,7 +9,7 @@ describe('Attaching tooltips to target elements in the DOM on each step', () =>
     cy.visit('/test/dummy/', {
       onLoad(contentWindow) {
         if (contentWindow.Shepherd) {
-          return Shepherd = contentWindow.Shepherd;
+          return (Shepherd = contentWindow.Shepherd);
```

Comment:
I'm curious, what are the parentheses for?

Reply:
I think this was added from the linter